### PR TITLE
PHP 8.1 compatibility

### PR DESCRIPTION
**Pull request by gquemener**

### CODE REVIEW

**Comment by netiul** [comment]
@prolic Can you have a look at this please?

File: composer.json [hunk 1]
```diff
@@ -75,6 +77,9 @@
         "test": "phpunit"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true
```
[the remark this answers is not in the document]
Comment (enumag) [reply composer.json:82]: This is normal @prolic. It's for Composer 2. ;-)

[the remark this answers is not in the document]
Comment (netiul) [reply composer.json:82]: It's kinda weird to add a deprecated package though 🤔

File: composer.json [hunk 2]
```diff
@@ -24,9 +24,11 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.1 || ^8.0",
+        "php": "^7.4 || ^8.0",
         "marc-mabe/php-enum": "^2.3.1 || ^3.0.0 || ^4.0.0",
-        "prooph/common": "^4.5.0"
+        "prooph/common": "^4.5.0",
+        "ramsey/uuid": "^4.3",
+        "ramsey/collection": "^1.2"
```
[the remark this answers is not in the document]
Comment (gquemener) [reply composer.json:31]: TBH, I can't recall. I've reverted it.

**Comment by gquemener** [comment]
Seems like you exceeded your plan on Travis
Screenshot from 2022-02-25 17-16-15

https://app.travis-ci.com/github/prooph/event-store/requests

Do you want to move to Github Actions?

**Comment by prolic** [comment]
Sure why not

**Comment by gquemener** [comment]
Build is passing, and available here: https://github.com/gquemener/event-store/actions/runs/1903613834.

I haven't setup integration with coveralls.io, as I don't have access to secrets.
Here's a documentation about it: https://github.com/marketplace/actions/coveralls-github-action

**Comment by gquemener** [comment]
Any blocker here? Build looks good to me (apart from the coveralls.io integration which could be done on a separate PR).

**Comment by netiul** [comment]
Ping @prolic. Could you have another look please? Thanks!

**Review by prolic — APPROVED** [review]

**Pull request merged**